Data/Ball.cs

```diff
@@ -0,0 +1,18 @@
+namespace Data
+{
+    public class Ball
+    {
+        public int X { get; set; }
+        public int Y { get; set; }
```

Comment:
I think that setters should be validated if you validate it in constructor

Reply:
There is  no validation in costructor. Validation is only in Logic Layer in `BallsManager` class